content/config_rb_backend.md

```diff
@@ -0,0 +1,290 @@
+    parent = "chef_infra/managing_chef_infra_server"
+    weight = 185
++++
+
+[\[edit on GitHub\]](https://github.com/chef/chef-web-docs/blob/master/content/config_rb_backend.md)
+
+
+## chef-backend.rb Options
+
+{{% config_rb_backend_summary %}}
+
+### Common 'Runit' flags for any backend service
+
+See <https://github.com/chef-cookbooks/runit> for details. Many of the
+flags are repeated across the various backend services - they are only
+documented once at the top here. The same defaults are used unless
+specified below.
+
+-   `postgresql.enable` Sets up and runs this service. `true` by
+    default.
+-   `postgresql.environment` A hash of environment variables with their
+    values as content used in the service's env directory.
+-   `postgresql.log_directory` The directory where the svlogd log
+    service will run. `'/var/log/chef-backend/postgresql/<version>'` by
+    default.
+-   `postgresql.log_rotation.file_maxbytes` The maximum size a log file
+    can grow to before it is automatically rotated. `104857600` by
+    default (100MB).
+-   `postgresql.log_rotation.num_to_keep` The maximum number of log
+    files that will be retained after rotation. `10` by default.
+-   `etcd.enable`
+-   `etcd.log_directory` `'/var/log/chef-backend/etcd'` by default
+-   `etcd.log_rotation.file_maxbytes`
+-   `etcd.log_rotation.num_to_keep`
+-   `elasticsearch.enable`
+-   `elasticsearch.log_directory`
+    `'/var/log/chef-backend/elasticsearch'` by default. Also affects
+    `path.logs` in the Elasticsearch configuration yml.
+-   `elasticsearch.log_rotation.file_maxbytes`
+-   `elasticsearch.log_rotation.num_to_keep`
+-   `leaderl.enable`
+-   `leaderl.log_directory` `'/var/log/chef-backend/leaderl'` by
+    default.
+-   `leaderl.start_down` Set the default state of the runit service to
+    'down' by creating \<sv_dir\>/down file. `true` by default.
+-   `leaderl.log_rotation.file_maxbytes`
+-   `leaderl.log_rotation.num_to_keep`
+
+### PostgreSQL settings
+
+-   `postgresql.db_superuser` Super user account to create. Password is
+    in chef-backend-secrets.json. `'chef_pgsql'` by default.
+-   `postgresql.md5_auth_cidr_addresses` A list of authorized addresses
+    from which other backend nodes can connect to perform streaming
+    replication. `samehost` and `samenet` are special symbols to allow
+    connections from the this node's IP address and its subnet. You may
+    also use `all` to match any IP address. You may specify a hostname
+    or IP address in CIDR format (`172.20.143.89/32` for a single host,
+    or `172.20.143.0/24` for a small network. See
+    <https://www.postgresql.org/docs/9.5/static/auth-pg-hba-conf.html>
+    for alternative formats. `["samehost", "samenet"]` by default.
+-   `postgresql.replication_user` Username used by postgres streaming
+    replicator when accessing this node. `'replicator'` by default.
+-   `postgresql.username` `'chef_pgsql'` by default.
```

Comment:
Is the `postgresql.username` the database administrator username? Something like:

`postgresql.username` Database administrator username, `'chef_pgsql'` by default.